CHANGELOG.md

```diff
@@ -41,6 +41,9 @@
 - Removed `driConfigureSupervisorCluster.ps1` from the \SampleScripts\ directory as functionality now provided using the `Invoke-DriDeployment` cmdlet.
 - Removed `driDeployTanzuCluster.ps1` from the \SampleScripts\ directory as functionality now provided using the `Invoke-DriDeployment` cmdlet.
 - Removed `driUndoDeployment.ps1` from the \SampleScripts\ directory as functionality now provided using the `Invoke-UndoDriDeployment` cmdlet.
+- Added `Invoke-GeneratePrivateKeyAndCsrFiles` cmdlet to generate private key and CSR files using Openssl, common name and input parameters.
```

Comment:
Why not `Invoke-GeneratePrivateKeyAndCsr`?

Reply:
addressed in commit 9d50a853ffce55094985dea3d9e658670a7ab67a